src/javascript/grid3_javascript_installation.md

```diff
@@ -1,66 +1,125 @@
-The [client](https://github.com/threefoldtech/tfgrid-sdk-ts/tree/development/packages/grid_client) is written using typescript to provide more convinient, type-checked code and it is used to deploy workloads like Virtual machines, kubernetes clusters, quantum storage, .. etc
+<h1>Installation</h1>
+
+<h2>Table of Contents</h2>
+
+- [Introduction](#introduction)
+- [Prerequisites](#prerequisites)
+- [Installation](#installation)
+  - [External Package](#external-package)
+  - [Local Usage](#local-usage)
+- [Getting Started](#getting-started)
+  - [Client Configuration](#client-configuration)
+- [Generate the Documentation](#generate-the-documentation)
+- [How to Run the Scripts](#how-to-run-the-scripts)
+- [Reference API](#reference-api)
+
+***
+
+## Introduction
+
+We present here the general steps required to install and use the ThreeFold Grid Client.
+
+The [Grid Client](https://github.com/threefoldtech/tfgrid-sdk-ts/tree/development/packages/grid_client) is written using [TypeScript](https://www.typescriptlang.org/) to provide more convenience and type-checked code. It is used to deploy workloads like virtual machines, kubernetes clusters, quantum storage, and more.
 
 ## Prerequisites
 
+To install the Grid Client, you will need the following on your machine:
+
 - node 16.13.1 or higher
 - npm 8.2.0 or higher
+- may need to install libtool (**apt-get install libtool**)
 
-> [nvm](https://nvm.sh/) is the recommended way for installing node.
+> Note: [nvm](https://nvm.sh/) is the recommended way for installing node.
+
+To use the Grid Client, you will need the following on the TFGrid:
+
+- An account
+- A twin
+- Mnemonics
+
+If it is not the case, please visit the [Get started section](../getstarted/tfgrid3_getstarted.md).
 
 ## Installation
 
-### External package
+### External Package
+
+To install the external package, simply run the following command:
 
 ```bash
 yarn add @threefold/grid_client
 ```
 
-### Local usage
+> Note: For the **qa**, **test** and **main** networks, please use @2.1.1 version.
+
+### Local Usage
+
+To use the Grid Client locally, clone the repository then install the Grid Client:
 
 - Clone the repository
+  - ```bash
+    git clone https://github.com/threefoldtech/tfgrid-sdk-ts
```

Comment:
Same here https://github.com/threefoldtech/tfgrid-sdk-ts#requirements

Reply:
I added Node.js ^18 in the Prerequisites section